package.json

```diff
@@ -40,6 +40,7 @@
     "fix": "eslint --fix '**/*.js'"
   },
   "dependencies": {
+    "chai": "^4.0.0",
```

Comment:
This should be in the `devDependencies` of `samples/package.json`, and shouldn't be here at all